<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerForm</class>
 <widget class="QMainWindow" name="TimerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tomato Timer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblCurrentTime">
      <property name="text">
       <string>&lt;b style=&quot;font-size:30pt&quot;&gt;00:00&lt;/b&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCurrentTask">
      <property name="toolTip">
       <string>Visual reminder of what you are currently working on</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>What are you working on?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="toolTip">
       <string>Start a 25 focus session</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuickBreak">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>take a 5 minute break</string>
      </property>
      <property name="text">
       <string>Quick Break</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLongBreak">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Take a 15 or 30 minute break after 4 focus sessions</string>
      </property>
      <property name="text">
       <string>Long Break</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
